main/defaults.h

```diff
@@ -38,4 +38,10 @@
 #define WAP_MAXCONN 4                       /*!< AP Maximum connections */
 #define WAP_AUTHMODE WIFI_AUTH_WPA_WPA2_PSK /*!< AP Authentication mode */
 
+#define USER_NAME_KEY "username"                   /*default user_name for app to do request to server*/
```

Comment:
```suggestion
#define USER_NAME_KEY "username"                   /*!< default user_name for app to do request to server*/
```

Use documentation style for comments.